modules/local/bam_rename.nf

```diff
@@ -1,10 +1,8 @@
 process BAM_RENAME {
```

Comment:
I think we can do better and use {meta.id} as a prefix in a previous process to rename the bam instead of using a process just for that.